tiered-storage/jcloud/src/test/java/org/apache/bookkeeper/mledger/offload/jcloud/BlobStoreTestBase.java

```diff
@@ -26,9 +28,12 @@
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
```

Comment:
Ivan, 

I definitely appreciate the amount of effort required to review the additional code changes that aren't directly related to the refactoring. My intent was to avoid introducing a "check-style bomb" that would create 100s of violations when someone attempted to build the entire code based and validate its style. I close to err on the side of caution and NOT submit code that I knew was bad. 

I can back out these changes if you prefer.

Reply:
New code should, of course, fit the checkstyle. The issue was in files that would otherwise be untouched.

For this patch, it's fine, there are other BC issues which are more important. The issue I have with unrelated changes in general though is that they make an already large patch bigger and that not only means it takes longer to review, but it takes longer to submit. I'd really love to do an analysis someday to see PR LOC vs time on the review queue. I had one myself recently that was at the 1500 mark, unfortunately, at it took 2.5 months to merge.